README.md

```diff
@@ -18,26 +48,96 @@ The application has the following runtime dependencies:
 * `SSO`
 * `USER_DETAILS`
 * `CUSTOMS_FINANCIALS_API`
-* `CUSTOMS_DATA_STORE`
 * `CUSTOMS_FINANCIALS_HODS_STUB`
 * `CUSTOMS_FINANCIALS_SDES_STUB`
-* `CUSTOMS_FINANCIALS_SESSION_CACHE`
 * `CONTACT_FRONTEND`
 
-## Development
+### Login enrolments
+
+The service can be accessed by using below enrolments and with below sample EORI numbers, via http://localhost:9949/auth-login-stub/gg-sign-in (on local) or https://<host:port>/auth-login-stub/gg-sign-in on DEV/QA/STAGING
+
+Redirect URL - `/customs/payment-records`
+
+| Enrolment Key	| Identifier Name | Identifier Value |
+| -------- | ------- | ------- |
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982000` |
+| `HMRC-CUS-ORG` | `EORINumber`| `GB744638982001` |
 
-You'll need [Service Manager](https://github.com/hmrc/service-manager) to develop locally.
+## Testing
 
-#### Service Manager Commands
+The minimum requirement for test coverage is 90%. Builds will fail when the project drops below this threshold.
 
-Start the required development services (make sure your service-manager-config folder is up to date)
+### Unit Tests
 
-    sm --start CUSTOMS_FINANCIALS_ALL -f
+| Command    | Description |
+| -------- | ------- |
+| `sbt test` | Runs unit tests locally |
+| `sbt "test:testOnly *TEST_FILE_NAME*"` | runs tests for a single file |
+
+### Coverage
+
+| Command    | Description |
+| -------- | ------- |
+| `sbt clean coverage test coverageReport` | Generates a unit test coverage report that you can find here target/scala-3.3.5/scoverage-report/index.html  |
+
+## Available Routes
+
+You can find a list of microservice specific routes here - `/conf/app.routes`
+
+Application entrypoint:  `/customs/payment-records` 
+
+| Path                                  |
+| ----                                  |
+| GET /requested/:fileRole              |
```

Comment:
I don't think we should bother with listing any routes on our frontends. Let's just point to app.routes.